apps/studio/components/interfaces/Auth/Policies/AIPolicyEditorPanel/QueryError.tsx

```diff
@@ -0,0 +1,48 @@
+import { QueryResponseError } from 'data/sql/execute-sql-mutation'
+import { AiIcon, AlertTitle_Shadcn_, Alert_Shadcn_, Button, IconAlertCircle } from 'ui'
+
+const QueryError = ({
+  error,
+  onSelectDebug,
+}: {
+  error: QueryResponseError
+  onSelectDebug: () => void
+}) => {
+  const formattedError =
+    (error?.formattedError?.split('\n') ?? [])?.filter((x: string) => x.length > 0) ?? []
+
+  return (
+    <div className="flex flex-col gap-y-3">
+      <Alert_Shadcn_
+        variant="warning"
+        className="flex items-center [&>svg]:relative [&>svg]:left-0 [&>svg]:top-0"
+      >
+        <IconAlertCircle strokeWidth={2} />
+        <AlertTitle_Shadcn_ className="m-0 !pl-4">Error running SQL query</AlertTitle_Shadcn_>
```

Comment:
not sure what the state of the responses is at the moment, but I'm getting this fyi

https://share.cleanshot.com/5fRfghyk
both the initial response and the debug response both have errors

Reply:
open ai output issue cc @ivasilov